tests/test_fields.py

```diff
@@ -14,6 +14,16 @@ def test_basic_field():
     assert field.type == 'type'
 
 
+def test_basic_field_defaults():
+    field = Field()
+
+    assert field.name == ''
+    assert field.type == Field
+    assert field.required
+    assert not field.nullable
+    assert not field.empty
```

Comment:
Maybe assert that the attributes are `False`, and souldn't be `None`

```suggestion
    assert field.required is True
    assert field.nullable is False
    assert field.empty is False
```